### PR TITLE
Adding GA and Angulartics

### PR DESCRIPTION
**Pull request by pstjean**

### CODE REVIEW

File: src/assets/google-analytics.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    m = s.getElementsByTagName(o)[0];
+  a.async = 1;
+  a.src = g;
+  m.parentNode.insertBefore(a, m);
+}
+
+if (window.ENV && window.ENV.GA_KEY) {
+  initGoogleAnalytics(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
+  ga('create', window.ENV.GA_KEY, 'auto');
+}
```
Comment (sandikbarr) [inline src/assets/google-analytics.js:16]: Does this need to get added to `index.pug`? Looks like we're including it there in Publish and Metrics https://github.com/PRX/publish.prx.org/blob/master/src/index.pug#L21

**Review by sandikbarr — CHANGES_REQUESTED** [review]
think you need to include that `google-analytics.js` files in `index.pug`

File: src/assets/google-analytics.js (hunk 1 above)
Comment (pstjean) [reply src/assets/google-analytics.js:16]: Yep, totally. I was thinking the PRX node server iterated through assets and did this automatically for some reason.

**Review by sandikbarr — APPROVED** [review]
👍

**Pull request merged**